langchain/retrievers/azure_cognitive_search.py

```diff
@@ -33,6 +33,8 @@ class AzureCognitiveSearchRetriever(BaseRetriever):
     """ClientSession, in case we want to reuse connection for better performance."""
     content_key: str = "content"
     """Key in a retrieved result to set as the Document page_content."""
+    top_n: Optional[int] = 50
```

Comment:
convention in other retrievers/vectorstores is to call this k or top_k, could we switch to that?

also for the sake of not unexpectedly changing default behavior for folks already using this, should we set default to None?

Reply:
Both changed!